lib/asciitable/table.go

```diff
@@ -41,6 +42,7 @@ type Table struct {
 	columns   []Column
 	rows      [][]string
 	footnotes map[string]string
+	mu        sync.Mutex
```

Comment:
Why do we need a mutex here?

Reply:
The table is first searched, and then manipulated. Calling DiscardEmpty() and Add() or AsBuffer() at the same time from different goroutines panics, or eats data.

Though, it would probably be a bug by library user anyway.